<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Comunidades de peces del Golfo de California</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="Report_files/libs/clipboard/clipboard.min.js"></script>
<script src="Report_files/libs/quarto-html/quarto.js"></script>
<script src="Report_files/libs/quarto-html/popper.min.js"></script>
<script src="Report_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Report_files/libs/quarto-html/anchor.min.js"></script>
<link href="Report_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Report_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Report_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Report_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Report_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

<script src="Report_files/libs/kePrint-0.0.1/kePrint.js"></script>
<link href="Report_files/libs/lightable-0.0.1/lightable.css" rel="stylesheet">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Comunidades de peces del Golfo de California</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="resumen-de-rendimiento-y-filtrado-de-lecturas" class="level3">
<h3 class="anchored" data-anchor-id="resumen-de-rendimiento-y-filtrado-de-lecturas">Resumen de rendimiento y filtrado de lecturas</h3>
<p>En este paso revisamos cuántas lecturas (secuencias) se obtuvieron y cómo se filtraron a lo largo del pipeline. El seguimiento del número de lecturas en cada etapa del pipeline nos permite evaluar la eficiencia y calidad del proceso de secuenciación y filtrado. Si se pierden muchas lecturas en algún paso, puede indicar problemas técnicos o errores en el procesamiento. Este control de calidad es esencial para asegurar que los resultados sean confiables y representativos de la comunidad biológica estudiada.</p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-condensed table-sm small" data-quarto-postprocess="true">
<caption>Resumen de rendimiento y filtrado de lecturas</caption>
<thead>
<tr class="header">
<th style="text-align: center;" data-quarto-table-cell-role="th">input</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">filtered</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">denoisedF</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">denoisedR</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">merged</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">nonchim</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">length_filter</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">3604247</td>
<td style="text-align: center;">3469483</td>
<td style="text-align: center;">3437368</td>
<td style="text-align: center;">3392900</td>
<td style="text-align: center;">3125684</td>
<td style="text-align: center;">2821509</td>
<td style="text-align: center;">2747102</td>
</tr>
</tbody>
</table>


</div>
</div>
</section>
<section id="carga-y-organización-de-la-metadata-de-muestras" class="level3">
<h3 class="anchored" data-anchor-id="carga-y-organización-de-la-metadata-de-muestras">Carga y organización de la metadata de muestras</h3>
<p>La metadata conecta los datos de secuenciación con la información ecológica y ambiental de cada muestra. Sin una metadata bien organizada, no podríamos relacionar los patrones de diversidad con factores como el sitio de muestreo, la fecha, o las condiciones ambientales. Esto es fundamental para interpretar los resultados en un contexto ecológico y para responder preguntas científicas relevantes.</p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-striped table-hover table-condensed table-sm small" data-quarto-postprocess="true">
<caption>Tabla de metadata de muestras</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th"></th>
<th style="text-align: center;" data-quarto-table-cell-role="th">site_id</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">name</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">lat</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">lon</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">date</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">12S-GC-001-d1_S1</td>
<td style="text-align: center;">POR</td>
<td style="text-align: center;">El Portugues</td>
<td style="text-align: center;">24.74921</td>
<td style="text-align: center;">110.6736</td>
<td style="text-align: center;">10/29/2016</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-002-d1_S2</td>
<td style="text-align: center;">BSS</td>
<td style="text-align: center;">Bajo Seco Sur (South San Francisquito Island)</td>
<td style="text-align: center;">24.80856</td>
<td style="text-align: center;">110.5252</td>
<td style="text-align: center;">10/29/2016</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-003-d1_S3</td>
<td style="text-align: center;">ANI</td>
<td style="text-align: center;">Isla Las Animas</td>
<td style="text-align: center;">25.11358</td>
<td style="text-align: center;">110.5078</td>
<td style="text-align: center;">10/30/2016</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-004-d1_S4</td>
<td style="text-align: center;">DIE</td>
<td style="text-align: center;">Isla San Dieguito (North)</td>
<td style="text-align: center;">25.20403</td>
<td style="text-align: center;">110.6949</td>
<td style="text-align: center;">10/30/2016</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-005-d1_S5</td>
<td style="text-align: center;">SCR</td>
<td style="text-align: center;">Isla Santa Cruz</td>
<td style="text-align: center;">25.31262</td>
<td style="text-align: center;">110.6894</td>
<td style="text-align: center;">10/30/2016</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-006-d1_S6</td>
<td style="text-align: center;">SMA</td>
<td style="text-align: center;">San Mateo (North)</td>
<td style="text-align: center;">25.37902</td>
<td style="text-align: center;">110.9927</td>
<td style="text-align: center;">10/31/2016</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-007-d1_S7</td>
<td style="text-align: center;">BSN</td>
<td style="text-align: center;">Bajo Seco Norte (South Catalana Island)</td>
<td style="text-align: center;">25.61154</td>
<td style="text-align: center;">110.7704</td>
<td style="text-align: center;">10/31/2016</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-008-d1_S8</td>
<td style="text-align: center;">CAT</td>
<td style="text-align: center;">Isla Catalana (North)</td>
<td style="text-align: center;">25.71534</td>
<td style="text-align: center;">110.7756</td>
<td style="text-align: center;">10/31/2016</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-009-d1_S9</td>
<td style="text-align: center;">MON</td>
<td style="text-align: center;">Isla Monserrat (Las Galeras)</td>
<td style="text-align: center;">25.74411</td>
<td style="text-align: center;">111.0493</td>
<td style="text-align: center;">11/01/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-010-d1_S10</td>
<td style="text-align: center;">DAN</td>
<td style="text-align: center;">Isla Danzante</td>
<td style="text-align: center;">25.81281</td>
<td style="text-align: center;">111.2553</td>
<td style="text-align: center;">11/01/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-011-d1_S11</td>
<td style="text-align: center;">CAR</td>
<td style="text-align: center;">Isla Carmen (Balandra)</td>
<td style="text-align: center;">26.01657</td>
<td style="text-align: center;">111.1691</td>
<td style="text-align: center;">11/02/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-012-d1_S12</td>
<td style="text-align: center;">COR</td>
<td style="text-align: center;">Isla Coronado (La Lobera)</td>
<td style="text-align: center;">26.17440</td>
<td style="text-align: center;">111.2617</td>
<td style="text-align: center;">11/02/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-014-d1_S14</td>
<td style="text-align: center;">PUL</td>
<td style="text-align: center;">Punta Pulpito</td>
<td style="text-align: center;">26.51492</td>
<td style="text-align: center;">111.4433</td>
<td style="text-align: center;">11/03/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-016-d1_S16</td>
<td style="text-align: center;">MAR</td>
<td style="text-align: center;">Isla San Marcos (North)</td>
<td style="text-align: center;">27.26016</td>
<td style="text-align: center;">112.0890</td>
<td style="text-align: center;">11/04/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-017-d1_S17</td>
<td style="text-align: center;">TOR</td>
<td style="text-align: center;">Isla Tortuga</td>
<td style="text-align: center;">27.43053</td>
<td style="text-align: center;">111.0617</td>
<td style="text-align: center;">11/04/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-018-d1_S18</td>
<td style="text-align: center;">NOL</td>
<td style="text-align: center;">Isla San Pedro Nolasco (Punta Chivato)</td>
<td style="text-align: center;">27.96585</td>
<td style="text-align: center;">111.3710</td>
<td style="text-align: center;">11/06/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-019-d1_S19</td>
<td style="text-align: center;">PMA</td>
<td style="text-align: center;">Isla San Pedro Martir (Punta Rabijunco)</td>
<td style="text-align: center;">28.38201</td>
<td style="text-align: center;">112.2959</td>
<td style="text-align: center;">11/07/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-021-d1_S21</td>
<td style="text-align: center;">LOR</td>
<td style="text-align: center;">Isla San Lorenzo (South)</td>
<td style="text-align: center;">28.58472</td>
<td style="text-align: center;">112.7592</td>
<td style="text-align: center;">11/08/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-022-d1_S22</td>
<td style="text-align: center;">IA1</td>
<td style="text-align: center;">Isla Angel de la Guarda (Puerto Refugio II)</td>
<td style="text-align: center;">29.55492</td>
<td style="text-align: center;">113.5585</td>
<td style="text-align: center;">11/09/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-023-d1_S23</td>
<td style="text-align: center;">IA2</td>
<td style="text-align: center;">Isla Angel de la Guarda (Puerto Refugio III)</td>
<td style="text-align: center;">29.55194</td>
<td style="text-align: center;">113.5144</td>
<td style="text-align: center;">11/09/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-027-d1_S27</td>
<td style="text-align: center;">PAT</td>
<td style="text-align: center;">Isla Patos</td>
<td style="text-align: center;">29.26635</td>
<td style="text-align: center;">112.4641</td>
<td style="text-align: center;">11/12/16</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-029-d1_S29</td>
<td style="text-align: center;">TIB</td>
<td style="text-align: center;">La Reina (Isla Tiburon)</td>
<td style="text-align: center;">29.06492</td>
<td style="text-align: center;">112.5058</td>
<td style="text-align: center;">11/12/16</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-031-d1_S31</td>
<td style="text-align: center;">EST</td>
<td style="text-align: center;">Isla San Esteban, El Canon</td>
<td style="text-align: center;">28.72152</td>
<td style="text-align: center;">112.6126</td>
<td style="text-align: center;">11/13/2016</td>
</tr>
<tr class="even">
<td style="text-align: left;">12S-GC-060-d1_S60</td>
<td style="text-align: center;">CTR-</td>
<td style="text-align: center;">Negative control pool</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
</tr>
<tr class="odd">
<td style="text-align: left;">12S-GC-013-d1_S13</td>
<td style="text-align: center;">MOCK</td>
<td style="text-align: center;">Mock comm. 25 fish</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
</tr>
</tbody>
</table>


</div>
</div>
</section>
<section id="preparación-de-la-tabla-de-abundancias-de-asvs" class="level3">
<h3 class="anchored" data-anchor-id="preparación-de-la-tabla-de-abundancias-de-asvs">Preparación de la tabla de abundancias de ASVs</h3>
<p>Las ASVs (Amplicon Sequence Variants) representan las secuencias únicas detectadas en el estudio. En esta sección preparamos una matriz que indica cuántas veces aparece cada ASV en cada muestra. Esta matriz es la base para todos los análisis de diversidad y composición de comunidades. La matriz de abundancias de ASVs es la base para todos los análisis de diversidad y composición de comunidades. Nos permite cuantificar la presencia y abundancia de cada variante de secuencia en cada muestra, lo que es esencial para comparar comunidades, calcular índices de diversidad y detectar patrones ecológicos.</p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-sm table-striped small" data-quarto-postprocess="true">
<caption>Primeros 10 renglones de la matriz de abundancias de ASVs</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th"></th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-001-d1_S1</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-002-d1_S2</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-003-d1_S3</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-004-d1_S4</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-005-d1_S5</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-006-d1_S6</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-007-d1_S7</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-008-d1_S8</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-009-d1_S9</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-010-d1_S10</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-011-d1_S11</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-012-d1_S12</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-013-d1_S13</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-014-d1_S14</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-016-d1_S16</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-017-d1_S17</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-018-d1_S18</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-019-d1_S19</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-021-d1_S21</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-022-d1_S22</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-023-d1_S23</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-027-d1_S27</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-029-d1_S29</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-031-d1_S31</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">12S-GC-060-d1_S60</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCTAAATATATTCTAACACCTTAACTTAAAACCTTTCACCAACAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: right;">73108</td>
<td style="text-align: right;">29026</td>
<td style="text-align: right;">26972</td>
<td style="text-align: right;">27749</td>
<td style="text-align: right;">22139</td>
<td style="text-align: right;">28956</td>
<td style="text-align: right;">11758</td>
<td style="text-align: right;">21256</td>
<td style="text-align: right;">25616</td>
<td style="text-align: right;">26380</td>
<td style="text-align: right;">15364</td>
<td style="text-align: right;">29576</td>
<td style="text-align: right;">472</td>
<td style="text-align: right;">18961</td>
<td style="text-align: right;">19550</td>
<td style="text-align: right;">27430</td>
<td style="text-align: right;">19812</td>
<td style="text-align: right;">18225</td>
<td style="text-align: right;">17424</td>
<td style="text-align: right;">16907</td>
<td style="text-align: right;">15717</td>
<td style="text-align: right;">23627</td>
<td style="text-align: right;">25054</td>
<td style="text-align: right;">19436</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCGCCCGTCACTCTCCCCAAGCTTACAACGCTAAGTACTTAAAACCCTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGAA</td>
<td style="text-align: right;">13581</td>
<td style="text-align: right;">9113</td>
<td style="text-align: right;">9687</td>
<td style="text-align: right;">10999</td>
<td style="text-align: right;">12242</td>
<td style="text-align: right;">10214</td>
<td style="text-align: right;">88</td>
<td style="text-align: right;">8914</td>
<td style="text-align: right;">9751</td>
<td style="text-align: right;">9738</td>
<td style="text-align: right;">12061</td>
<td style="text-align: right;">10222</td>
<td style="text-align: right;">211</td>
<td style="text-align: right;">8090</td>
<td style="text-align: right;">7694</td>
<td style="text-align: right;">9876</td>
<td style="text-align: right;">8012</td>
<td style="text-align: right;">9681</td>
<td style="text-align: right;">8615</td>
<td style="text-align: right;">7363</td>
<td style="text-align: right;">6624</td>
<td style="text-align: right;">9960</td>
<td style="text-align: right;">8982</td>
<td style="text-align: right;">7498</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTTACTTACTCTAAGTACTTAAAACCTTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: right;">9721</td>
<td style="text-align: right;">8064</td>
<td style="text-align: right;">8160</td>
<td style="text-align: right;">10297</td>
<td style="text-align: right;">8527</td>
<td style="text-align: right;">8399</td>
<td style="text-align: right;">71</td>
<td style="text-align: right;">7790</td>
<td style="text-align: right;">7029</td>
<td style="text-align: right;">8235</td>
<td style="text-align: right;">8230</td>
<td style="text-align: right;">8924</td>
<td style="text-align: right;">18546</td>
<td style="text-align: right;">6153</td>
<td style="text-align: right;">6281</td>
<td style="text-align: right;">9414</td>
<td style="text-align: right;">7316</td>
<td style="text-align: right;">7983</td>
<td style="text-align: right;">6672</td>
<td style="text-align: right;">6779</td>
<td style="text-align: right;">5264</td>
<td style="text-align: right;">7936</td>
<td style="text-align: right;">7008</td>
<td style="text-align: right;">6689</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCGCCCGTCACTCTCCCCAAGCTTACCAATTTATATATCTAAAACGCTTTAACTGCGAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGAA</td>
<td style="text-align: right;">10666</td>
<td style="text-align: right;">10482</td>
<td style="text-align: right;">9276</td>
<td style="text-align: right;">8477</td>
<td style="text-align: right;">7950</td>
<td style="text-align: right;">11127</td>
<td style="text-align: right;">123</td>
<td style="text-align: right;">8580</td>
<td style="text-align: right;">9050</td>
<td style="text-align: right;">10115</td>
<td style="text-align: right;">8404</td>
<td style="text-align: right;">10631</td>
<td style="text-align: right;">191</td>
<td style="text-align: right;">5587</td>
<td style="text-align: right;">5620</td>
<td style="text-align: right;">10405</td>
<td style="text-align: right;">6463</td>
<td style="text-align: right;">7112</td>
<td style="text-align: right;">6269</td>
<td style="text-align: right;">5515</td>
<td style="text-align: right;">4473</td>
<td style="text-align: right;">7053</td>
<td style="text-align: right;">10050</td>
<td style="text-align: right;">13441</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCCAAATCTGTGTAAACACCTATATATAAACCCCTATAATTTTAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: right;">5952</td>
<td style="text-align: right;">12489</td>
<td style="text-align: right;">12042</td>
<td style="text-align: right;">7530</td>
<td style="text-align: right;">4838</td>
<td style="text-align: right;">11551</td>
<td style="text-align: right;">110</td>
<td style="text-align: right;">7632</td>
<td style="text-align: right;">10295</td>
<td style="text-align: right;">14349</td>
<td style="text-align: right;">8577</td>
<td style="text-align: right;">11475</td>
<td style="text-align: right;">4299</td>
<td style="text-align: right;">5897</td>
<td style="text-align: right;">5812</td>
<td style="text-align: right;">12011</td>
<td style="text-align: right;">7916</td>
<td style="text-align: right;">2786</td>
<td style="text-align: right;">4674</td>
<td style="text-align: right;">1762</td>
<td style="text-align: right;">2735</td>
<td style="text-align: right;">8168</td>
<td style="text-align: right;">10650</td>
<td style="text-align: right;">6052</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="even">
<td style="text-align: left;">CGCCCGTCACTCTCCCCGAGCAGAAAAATTCTTTACCTAAAAAGCCAAATAAATGTAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: right;">7872</td>
<td style="text-align: right;">10515</td>
<td style="text-align: right;">11452</td>
<td style="text-align: right;">8477</td>
<td style="text-align: right;">5324</td>
<td style="text-align: right;">10334</td>
<td style="text-align: right;">109</td>
<td style="text-align: right;">5771</td>
<td style="text-align: right;">9102</td>
<td style="text-align: right;">11117</td>
<td style="text-align: right;">4730</td>
<td style="text-align: right;">8979</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">5469</td>
<td style="text-align: right;">5338</td>
<td style="text-align: right;">9396</td>
<td style="text-align: right;">6962</td>
<td style="text-align: right;">4481</td>
<td style="text-align: right;">6419</td>
<td style="text-align: right;">3955</td>
<td style="text-align: right;">4276</td>
<td style="text-align: right;">7038</td>
<td style="text-align: right;">7764</td>
<td style="text-align: right;">18003</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCCCACTAAAAACATGTAACTAAACCTTAACACACGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: right;">10784</td>
<td style="text-align: right;">6569</td>
<td style="text-align: right;">6051</td>
<td style="text-align: right;">9322</td>
<td style="text-align: right;">11029</td>
<td style="text-align: right;">7450</td>
<td style="text-align: right;">75</td>
<td style="text-align: right;">7095</td>
<td style="text-align: right;">6428</td>
<td style="text-align: right;">7066</td>
<td style="text-align: right;">8122</td>
<td style="text-align: right;">6976</td>
<td style="text-align: right;">35</td>
<td style="text-align: right;">5407</td>
<td style="text-align: right;">5570</td>
<td style="text-align: right;">6788</td>
<td style="text-align: right;">6183</td>
<td style="text-align: right;">9221</td>
<td style="text-align: right;">9857</td>
<td style="text-align: right;">5730</td>
<td style="text-align: right;">4657</td>
<td style="text-align: right;">6023</td>
<td style="text-align: right;">7350</td>
<td style="text-align: right;">5689</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="even">
<td style="text-align: left;">GCCCGTCACTCTCCCCGAGCTCCCCAGACCATTTTTAACTAAAGCCCTGCAACTGCAAAGGAGAGGAAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: right;">16996</td>
<td style="text-align: right;">8379</td>
<td style="text-align: right;">7954</td>
<td style="text-align: right;">6479</td>
<td style="text-align: right;">4868</td>
<td style="text-align: right;">7336</td>
<td style="text-align: right;">3116</td>
<td style="text-align: right;">5351</td>
<td style="text-align: right;">7231</td>
<td style="text-align: right;">8141</td>
<td style="text-align: right;">3842</td>
<td style="text-align: right;">6981</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">3950</td>
<td style="text-align: right;">4078</td>
<td style="text-align: right;">6456</td>
<td style="text-align: right;">5194</td>
<td style="text-align: right;">4102</td>
<td style="text-align: right;">4135</td>
<td style="text-align: right;">2548</td>
<td style="text-align: right;">3292</td>
<td style="text-align: right;">6572</td>
<td style="text-align: right;">8004</td>
<td style="text-align: right;">3692</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCCGAGCTTACTAACTCTGTAATACTTAAAACCTTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: right;">5060</td>
<td style="text-align: right;">4602</td>
<td style="text-align: right;">4744</td>
<td style="text-align: right;">4719</td>
<td style="text-align: right;">3396</td>
<td style="text-align: right;">4015</td>
<td style="text-align: right;">37</td>
<td style="text-align: right;">3216</td>
<td style="text-align: right;">3630</td>
<td style="text-align: right;">4280</td>
<td style="text-align: right;">2300</td>
<td style="text-align: right;">4219</td>
<td style="text-align: right;">77</td>
<td style="text-align: right;">2433</td>
<td style="text-align: right;">3098</td>
<td style="text-align: right;">4057</td>
<td style="text-align: right;">2920</td>
<td style="text-align: right;">2273</td>
<td style="text-align: right;">3066</td>
<td style="text-align: right;">2264</td>
<td style="text-align: right;">2915</td>
<td style="text-align: right;">2261</td>
<td style="text-align: right;">3746</td>
<td style="text-align: right;">6166</td>
<td style="text-align: right;">0</td>
</tr>
<tr class="even">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTCACCTAACAACTATGCCTAAAACACAAAAAATGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: right;">5418</td>
<td style="text-align: right;">3810</td>
<td style="text-align: right;">2718</td>
<td style="text-align: right;">3468</td>
<td style="text-align: right;">4682</td>
<td style="text-align: right;">4370</td>
<td style="text-align: right;">0</td>
<td style="text-align: right;">3881</td>
<td style="text-align: right;">3613</td>
<td style="text-align: right;">3496</td>
<td style="text-align: right;">3925</td>
<td style="text-align: right;">4041</td>
<td style="text-align: right;">481</td>
<td style="text-align: right;">3408</td>
<td style="text-align: right;">2406</td>
<td style="text-align: right;">4186</td>
<td style="text-align: right;">3015</td>
<td style="text-align: right;">2741</td>
<td style="text-align: right;">2928</td>
<td style="text-align: right;">3132</td>
<td style="text-align: right;">3715</td>
<td style="text-align: right;">3947</td>
<td style="text-align: right;">3671</td>
<td style="text-align: right;">2437</td>
<td style="text-align: right;">0</td>
</tr>
</tbody>
</table>


</div>
</div>
</section>
<section id="preparación-de-la-tabla-taxonómica" class="level3">
<h3 class="anchored" data-anchor-id="preparación-de-la-tabla-taxonómica">Preparación de la tabla taxonómica</h3>
<p>La tabla taxonómica nos permite traducir las secuencias en información biológica significativa, identificando los organismos presentes en las muestras. Esto es crucial para interpretar los resultados en términos de biodiversidad y para realizar análisis ecológicos a diferentes niveles taxonómicos (especie, género, familia, etc.).</p>
<div class="cell">
<div class="cell-output-display">
<table class="table table-sm table-striped small" data-quarto-postprocess="true">
<caption>Primeros 10 renglones de la tabla taxonómica</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th"></th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Kingdom</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Phylum</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Class</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Order</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Family</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Genus</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Species</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCTAAATATATTCTAACACCTTAACTTAAAACCTTTCACCAACAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Perciformes</td>
<td style="text-align: center;">Serranidae</td>
<td style="text-align: center;">Mycteroperca</td>
<td style="text-align: center;">Mycteroperca rosacea</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCGCCCGTCACTCTCCCCAAGCTTACAACGCTAAGTACTTAAAACCCTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGAA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Lutjanidae</td>
<td style="text-align: center;">Lutjanus</td>
<td style="text-align: center;">Lutjanus peru</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTTACTTACTCTAAGTACTTAAAACCTTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Lutjanidae</td>
<td style="text-align: center;">Lutjanus</td>
<td style="text-align: center;">Lutjanus argentiventris</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCGCCCGTCACTCTCCCCAAGCTTACCAATTTATATATCTAAAACGCTTTAACTGCGAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGAA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Scombriformes</td>
<td style="text-align: center;">Scombridae</td>
<td style="text-align: center;">Thunnus</td>
<td style="text-align: center;">Thunnus albacares</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCCAAATCTGTGTAAACACCTATATATAAACCCCTATAATTTTAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Perciformes</td>
<td style="text-align: center;">Serranidae</td>
<td style="text-align: center;">Paralabrax</td>
<td style="text-align: center;">Paralabrax nebulifer</td>
</tr>
<tr class="even">
<td style="text-align: left;">CGCCCGTCACTCTCCCCGAGCAGAAAAATTCTTTACCTAAAAAGCCAAATAAATGTAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Labriformes</td>
<td style="text-align: center;">Labridae</td>
<td style="text-align: center;">Scarus</td>
<td style="text-align: center;">Scarus perrico</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCCCACTAAAAACATGTAACTAAACCTTAACACACGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
<td style="text-align: center;">NA</td>
</tr>
<tr class="even">
<td style="text-align: left;">GCCCGTCACTCTCCCCGAGCTCCCCAGACCATTTTTAACTAAAGCCCTGCAACTGCAAAGGAGAGGAAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Carangiformes</td>
<td style="text-align: center;">Carangidae</td>
<td style="text-align: center;">Seriola</td>
<td style="text-align: center;">Seriola quinqueradiata</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCCGAGCTTACTAACTCTGTAATACTTAAAACCTTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Lutjanidae</td>
<td style="text-align: center;">Lutjanus</td>
<td style="text-align: center;">Lutjanus novemfasciatus</td>
</tr>
<tr class="even">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTCACCTAACAACTATGCCTAAAACACAAAAAATGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Haemulidae</td>
<td style="text-align: center;">Haemulon</td>
<td style="text-align: center;">Haemulon sexfasciatum</td>
</tr>
</tbody>
</table>


</div>
</div>
</section>
<section id="curación-geográfica-de-especies-de-peces" class="level3">
<h3 class="anchored" data-anchor-id="curación-geográfica-de-especies-de-peces">Curación geográfica de especies de peces</h3>
<p>En este paso filtramos y corregimos la asignación taxonómica para asegurarnos de que solo se incluyan especies de peces relevantes para el área de estudio. Esto evita sesgos en los resultados y asegura que las conclusiones sean aplicables a la comunidad local. Además, elimina posibles contaminantes o errores de asignación que podrían distorsionar el análisis.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>fish_sp <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">"../metadata/Fish_speciesGC.csv"</span>, <span class="at">row.names =</span> <span class="cn">NULL</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>taxa2 <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(taxa_df)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>fish_species <span class="ot">&lt;-</span> fish_sp<span class="sc">$</span>Species2</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>check_and_replace <span class="ot">&lt;-</span> <span class="cf">function</span>(species, genus, family, order) {</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Cambios específicos de especie y género</span></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(genus) <span class="sc">&amp;&amp;</span> genus <span class="sc">==</span> <span class="st">"Balistes"</span>) species <span class="ot">&lt;-</span> <span class="st">"Balistes polylepis"</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(genus) <span class="sc">&amp;&amp;</span> genus <span class="sc">==</span> <span class="st">"Atractoscion"</span>) species <span class="ot">&lt;-</span> <span class="st">"Atractoscion nobilis"</span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(genus) <span class="sc">&amp;&amp;</span> genus <span class="sc">==</span> <span class="st">"Elops"</span>) species <span class="ot">&lt;-</span> <span class="st">"Elops affinis"</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(genus) <span class="sc">&amp;&amp;</span> genus <span class="sc">==</span> <span class="st">"Kajikia"</span>) species <span class="ot">&lt;-</span> <span class="st">"Kajikia audax"</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(genus) <span class="sc">&amp;&amp;</span> genus <span class="sc">==</span> <span class="st">"Prionurus"</span>) species <span class="ot">&lt;-</span> <span class="st">"Prionurus laticlavius"</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Cambios específicos de familia y orden</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(family) <span class="sc">&amp;&amp;</span> family <span class="sc">==</span> <span class="st">"Pomacentridae"</span>)   order <span class="ot">&lt;-</span> <span class="st">"Blenniiformes"</span></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(family) <span class="sc">&amp;&amp;</span> family <span class="sc">==</span> <span class="st">"Sphyraenidae"</span>)    order <span class="ot">&lt;-</span> <span class="st">"Carangiformes"</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(family) <span class="sc">&amp;&amp;</span> family <span class="sc">==</span> <span class="st">"Sciaenidae"</span>)      order <span class="ot">&lt;-</span> <span class="st">"Perciformes"</span></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(family) <span class="sc">&amp;&amp;</span> family <span class="sc">==</span> <span class="st">"Malacanthidae"</span>)   order <span class="ot">&lt;-</span> <span class="st">"Perciformes"</span></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (<span class="sc">!</span><span class="fu">is.na</span>(family) <span class="sc">&amp;&amp;</span> family <span class="sc">==</span> <span class="st">"Pomacanthidae"</span>)   order <span class="ot">&lt;-</span> <span class="st">"Acanthuriformes"</span></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(<span class="fu">list</span>(<span class="at">species =</span> species, <span class="at">order =</span> order))</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Aplica la función y actualiza Species y Order en taxa2</span></span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a>res <span class="ot">&lt;-</span> <span class="fu">mapply</span>(check_and_replace,</span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a>              <span class="at">species =</span> taxa2[, <span class="st">"Species"</span>],</span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a>              <span class="at">genus   =</span> taxa2[, <span class="st">"Genus"</span>],</span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a>              <span class="at">family  =</span> taxa2[, <span class="st">"Family"</span>],</span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a>              <span class="at">order   =</span> taxa2[, <span class="st">"Order"</span>],</span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a>              <span class="at">SIMPLIFY =</span> <span class="cn">FALSE</span>)</span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a>taxa2<span class="sc">$</span>Species <span class="ot">&lt;-</span> <span class="fu">sapply</span>(res, <span class="cf">function</span>(x) x<span class="sc">$</span>species)</span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a>taxa2<span class="sc">$</span>Order   <span class="ot">&lt;-</span> <span class="fu">sapply</span>(res, <span class="cf">function</span>(x) x<span class="sc">$</span>order)</span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a>taxa2 <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(taxa2)</span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a>taxa2_filtered <span class="ot">&lt;-</span> dplyr<span class="sc">::</span><span class="fu">filter</span>(taxa2, <span class="sc">!</span>(Species <span class="sc">==</span> <span class="st">"Eukaryotic synthetic"</span> <span class="sc">|</span> <span class="fu">grepl</span>(<span class="st">"^NA sp$"</span>, Species)))</span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir a matriz para tax_table, conservando rownames (secuencias)</span></span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a>taxa2 <span class="ot">&lt;-</span> <span class="fu">as.matrix</span>(taxa2_filtered)</span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>taxa2[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>, <span class="dv">1</span><span class="sc">:</span><span class="dv">7</span>] <span class="sc">%&gt;%</span></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable</span>(<span class="at">format =</span> <span class="st">"html"</span>, <span class="at">caption =</span> <span class="st">"Primeros 10 renglones de la tabla taxonómica curada"</span>, <span class="at">align =</span> <span class="st">"c"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">full_width =</span> <span class="cn">TRUE</span>, <span class="at">font_size =</span> <span class="dv">10</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<table class="table table-sm table-striped small" data-quarto-postprocess="true">
<caption>Primeros 10 renglones de la tabla taxonómica curada</caption>
<thead>
<tr class="header">
<th style="text-align: left;" data-quarto-table-cell-role="th"></th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Kingdom</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Phylum</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Class</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Order</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Family</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Genus</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">Species</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCTAAATATATTCTAACACCTTAACTTAAAACCTTTCACCAACAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Perciformes</td>
<td style="text-align: center;">Serranidae</td>
<td style="text-align: center;">Mycteroperca</td>
<td style="text-align: center;">Mycteroperca rosacea</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCGCCCGTCACTCTCCCCAAGCTTACAACGCTAAGTACTTAAAACCCTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGAA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Lutjanidae</td>
<td style="text-align: center;">Lutjanus</td>
<td style="text-align: center;">Lutjanus peru</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTTACTTACTCTAAGTACTTAAAACCTTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Lutjanidae</td>
<td style="text-align: center;">Lutjanus</td>
<td style="text-align: center;">Lutjanus argentiventris</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCGCCCGTCACTCTCCCCAAGCTTACCAATTTATATATCTAAAACGCTTTAACTGCGAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGAA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Scombriformes</td>
<td style="text-align: center;">Scombridae</td>
<td style="text-align: center;">Thunnus</td>
<td style="text-align: center;">Thunnus albacares</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCCAAATCTGTGTAAACACCTATATATAAACCCCTATAATTTTAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Perciformes</td>
<td style="text-align: center;">Serranidae</td>
<td style="text-align: center;">Paralabrax</td>
<td style="text-align: center;">Paralabrax nebulifer</td>
</tr>
<tr class="even">
<td style="text-align: left;">CGCCCGTCACTCTCCCCGAGCAGAAAAATTCTTTACCTAAAAAGCCAAATAAATGTAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Labriformes</td>
<td style="text-align: center;">Labridae</td>
<td style="text-align: center;">Scarus</td>
<td style="text-align: center;">Scarus perrico</td>
</tr>
<tr class="odd">
<td style="text-align: left;">GCCCGTCACTCTCCCCGAGCTCCCCAGACCATTTTTAACTAAAGCCCTGCAACTGCAAAGGAGAGGAAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Carangiformes</td>
<td style="text-align: center;">Carangidae</td>
<td style="text-align: center;">Seriola</td>
<td style="text-align: center;">Seriola quinqueradiata</td>
</tr>
<tr class="even">
<td style="text-align: left;">GCCCGTCACTCTCCCCGAGCTTACTAACTCTGTAATACTTAAAACCTTAAAACTGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGG</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Lutjanidae</td>
<td style="text-align: center;">Lutjanus</td>
<td style="text-align: center;">Lutjanus novemfasciatus</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTCACCTAACAACTATGCCTAAAACACAAAAAATGCAAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Lutjaniformes</td>
<td style="text-align: center;">Haemulidae</td>
<td style="text-align: center;">Haemulon</td>
<td style="text-align: center;">Haemulon sexfasciatum</td>
</tr>
<tr class="even">
<td style="text-align: left;">CGCCCGTCACTCTCCCCAAGCTTCTTTTGACTTAATAATTAATATACTAATTCTGCTAAGGGGAGGCAAGTCGTAACATGGTAAGTGTACCGGA</td>
<td style="text-align: center;">Metazoa</td>
<td style="text-align: center;">Chordata</td>
<td style="text-align: center;">Actinopteri</td>
<td style="text-align: center;">Tetraodontiformes</td>
<td style="text-align: center;">Balistidae</td>
<td style="text-align: center;">Balistes</td>
<td style="text-align: center;">Balistes polylepis</td>
</tr>
</tbody>
</table>


</div>
</div>
</section>
<section id="creación-del-objeto-phyloseq-y-asignación-de-secuencias" class="level3">
<h3 class="anchored" data-anchor-id="creación-del-objeto-phyloseq-y-asignación-de-secuencias">Creación del objeto <em>phyloseq</em> y asignación de secuencias</h3>
<p>Integrar todos los datos en un objeto <em>phyloseq</em> facilita el manejo y análisis de datos complejos, permitiendo realizar análisis multivariados, visualizaciones y filtrados de manera eficiente y reproducible. Esto es clave para mantener la integridad y trazabilidad de los datos a lo largo del análisis.</p>
<ol type="1">
<li><p>Crea la matriz de abundancias (<code>otu_table</code>) Cada fila representa una secuencia (ASV) y cada columna una muestra. Aquí se almacena cuántas veces aparece cada ASV en cada muestra.</p></li>
<li><p>Crea la tabla taxonómica (<code>tax_table</code>) Relaciona cada ASV con su clasificación biológica (reino, filo, clase, orden, familia, género, especie).</p></li>
<li><p>Agrega la metadata de las muestras (<code>sample_data</code>) Incluye información ambiental y de muestreo para cada muestra (sitio, fecha, coordenadas, etc.).</p></li>
<li><p>Integra todo en un objeto <em>phyloseq</em> El objeto phyloseq permite manejar todos estos datos juntos, facilitando análisis de diversidad, visualizaciones y filtrados.</p></li>
<li><p>Asigna las secuencias originales y renombra los ASVs Guarda las secuencias originales en el objeto y les da nombres estandarizados (ASV1, ASV2, …).</p></li>
</ol>
<div class="cell">
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a>ASV  <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">otu_table</span>(ASV_mat, <span class="at">taxa_are_rows =</span> <span class="cn">TRUE</span>)</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>TAXA <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">tax_table</span>(taxa2)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>MD   <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">sample_data</span>(md)</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>ps   <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">phyloseq</span>(ASV, TAXA, MD)</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Guardar secuencias originales en slot refseq y renombrar ASVs a ASV1..n</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> Biostrings<span class="sc">::</span><span class="fu">DNAStringSet</span>(phyloseq<span class="sc">::</span><span class="fu">taxa_names</span>(ps))</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(dna) <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">taxa_names</span>(ps)</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>ps <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">merge_phyloseq</span>(ps, dna)</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>phyloseq<span class="sc">::</span><span class="fu">taxa_names</span>(ps) <span class="ot">&lt;-</span> <span class="fu">paste0</span>(<span class="st">"ASV"</span>, <span class="fu">seq_len</span>(phyloseq<span class="sc">::</span><span class="fu">ntaxa</span>(ps)))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<ul>
<li><p><code>ps</code> muestra un resumen del objeto phyloseq, incluyendo el número de ASVs, muestras y taxones.</p></li>
<li><p><code>otu_table(ps)</code> accede a la matriz de abundancias.</p></li>
<li><p><code>tax_table(ps)</code> accede a la tabla taxonómica.</p></li>
<li><p><code>sample_data(ps)</code> accede a la metadata de las muestras.</p></li>
<li><p><code>refseq(ps)</code> accede a las secuencias originales asociadas a cada ASV.</p></li>
</ul>
</section>
<section id="agrupación-de-asvs-por-especie-y-cálculo-de-abundancias-relativas" class="level3">
<h3 class="anchored" data-anchor-id="agrupación-de-asvs-por-especie-y-cálculo-de-abundancias-relativas">Agrupación de ASVs por especie y cálculo de abundancias relativas</h3>
<p>Agrupamos las ASVs por especie para simplificar el análisis y calcular la abundancia relativa de cada especie en cada muestra. Esto nos permite comparar la composición de especies entre sitios y entender la estructura de las comunidades.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># merge asvs by species</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>ps.sp <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">tax_glom</span>(ps, <span class="st">"Species"</span>)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Crear versión proporcional</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>ps.sp.prop <span class="ot">&lt;-</span> phyloseq<span class="sc">::</span><span class="fu">transform_sample_counts</span>(ps.sp, <span class="cf">function</span>(x) x<span class="sc">/</span><span class="fu">sum</span>(x))</span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>check_df <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(<span class="fu">tax_table</span>(ps.sp.prop))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="generación-de-la-tabla-de-abundancias-por-especie-y-muestra" class="level3">
<h3 class="anchored" data-anchor-id="generación-de-la-tabla-de-abundancias-por-especie-y-muestra">Generación de la tabla de abundancias por especie y muestra</h3>
<p>Creamos una tabla que muestra cuántas lecturas corresponden a cada especie en cada muestra. Esta tabla es útil para análisis personalizados y para exportar los resultados a otros formatos o programas.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>ASV.df<span class="ot">&lt;-</span><span class="fu">as.data.frame</span>(ASV)<span class="sc">%&gt;%</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="at">var=</span><span class="st">"seqID"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>  tidyr<span class="sc">::</span><span class="fu">pivot_longer</span>(<span class="at">cols=</span><span class="sc">-</span>seqID,<span class="at">names_to =</span> <span class="st">"Sample_name"</span>, <span class="at">values_to =</span> <span class="st">"nReads"</span>)</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>TAXA.df<span class="ot">&lt;-</span><span class="fu">as.data.frame</span>(TAXA)<span class="sc">%&gt;%</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="at">var=</span><span class="st">"seqID"</span>)</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>species.site.reads <span class="ot">&lt;-</span> ASV.df <span class="sc">%&gt;%</span> </span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(TAXA.df, <span class="at">by=</span><span class="st">"seqID"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(Species)) <span class="sc">%&gt;%</span> </span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample_name, Species)</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>species.site.reads <span class="ot">&lt;-</span> <span class="fu">aggregate</span>(nReads <span class="sc">~</span> Sample_name <span class="sc">+</span> Species, <span class="at">data =</span> species.site.reads, sum)</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>species.site.reads <span class="ot">&lt;-</span> species.site.reads<span class="sc">%&gt;%</span> </span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>   <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> Sample_name, <span class="at">values_from =</span> nReads, <span class="at">values_fill =</span> <span class="dv">0</span>) </span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-17"><a href="#cb4-17" aria-hidden="true" tabindex="-1"></a><span class="fu">write.csv</span>(species.site.reads, <span class="st">"../final_data/csv_output/taxon_table.csv"</span>)</span>
<span id="cb4-18"><a href="#cb4-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-19"><a href="#cb4-19" aria-hidden="true" tabindex="-1"></a>species.site.reads[<span class="dv">1</span><span class="sc">:</span><span class="dv">10</span>, <span class="dv">1</span><span class="sc">:</span><span class="dv">3</span>] <span class="sc">%&gt;%</span></span>
<span id="cb4-20"><a href="#cb4-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable</span>(<span class="at">format =</span> <span class="st">"html"</span>, <span class="at">caption =</span> <span class="st">"Primeros 10 renglones y 3 columnas de la tabla de abundancias por especie y muestra"</span>, <span class="at">align =</span> <span class="st">"c"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb4-21"><a href="#cb4-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">full_width =</span> <span class="cn">TRUE</span>, <span class="at">font_size =</span> <span class="dv">10</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<table class="table table-sm table-striped small" data-quarto-postprocess="true">
<caption>Primeros 10 renglones y 3 columnas de la tabla de abundancias por especie y muestra</caption>
<thead>
<tr class="header">
<th style="text-align: center;" data-quarto-table-cell-role="th">Species</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">12S-GC-001-d1_S1</th>
<th style="text-align: center;" data-quarto-table-cell-role="th">12S-GC-002-d1_S2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Abudefduf troschelii</td>
<td style="text-align: center;">968</td>
<td style="text-align: center;">946</td>
</tr>
<tr class="even">
<td style="text-align: center;">Acanthurus mata</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Acanthurus xanthopterus</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="even">
<td style="text-align: center;">Anisotremus interruptus</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Atractoscion nobilis</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="even">
<td style="text-align: center;">Balistes polylepis</td>
<td style="text-align: center;">1475</td>
<td style="text-align: center;">3939</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Bodianus rufus</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="even">
<td style="text-align: center;">Calamus arctifrons</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">0</td>
</tr>
<tr class="odd">
<td style="text-align: center;">Caulolatilus microps</td>
<td style="text-align: center;">0</td>
<td style="text-align: center;">128</td>
</tr>
<tr class="even">
<td style="text-align: center;">Cephalopholis panamensis</td>
<td style="text-align: center;">100</td>
<td style="text-align: center;">54</td>
</tr>
</tbody>
</table>


</div>
</div>
</section>
<section id="visualización-de-la-distribución-de-longitudes-de-secuencias" class="level3">
<h3 class="anchored" data-anchor-id="visualización-de-la-distribución-de-longitudes-de-secuencias">Visualización de la distribución de longitudes de secuencias</h3>
<p>Analizar la longitud de las secuencias nos ayuda a verificar la calidad y consistencia de los datos. Secuencias demasiado cortas o largas pueden indicar problemas en la amplificación o secuenciación.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Visualización%20de%20la%20distribución%20de%20longitudes%20de%20secuencias-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="curva-de-acumulación-de-especies" class="level3">
<h3 class="anchored" data-anchor-id="curva-de-acumulación-de-especies">Curva de acumulación de especies</h3>
<p>La curva de acumulación muestra cómo aumenta el número de especies detectadas a medida que se analizan más muestras. Es útil para evaluar si el muestreo fue suficiente y estimar la riqueza total de especies en el área de estudio.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Curva%20de%20acumulación%20de%20especies-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="abundancia-relativa-nivel-familia-barras-apiladas" class="level3">
<h3 class="anchored" data-anchor-id="abundancia-relativa-nivel-familia-barras-apiladas">Abundancia relativa, nivel Familia, barras apiladas</h3>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Abundancia%20relativa,%20nivel%20Familia,%20barras%20apiladas-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="top-5-familias-barras-apiladas" class="level3">
<h3 class="anchored" data-anchor-id="top-5-familias-barras-apiladas">Top 5 Familias, barras apiladas</h3>
<p>Analizar la composición de especies dentro de las familias más abundantes permite entender la diversidad interna de cada grupo y detectar especies clave o raras. Es útil para estudios de conservación y para identificar especies indicadoras.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Top%205%20Familias-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="abundancia-relativa-nivel-especie-barras-apiladas" class="level3">
<h3 class="anchored" data-anchor-id="abundancia-relativa-nivel-especie-barras-apiladas">Abundancia relativa, nivel Especie, barras apiladas</h3>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Abundancia%20relativa,%20nivel%20Especie,%20barras%20apiladas-1.png" class="img-fluid" width="1152"></p>
</div>
</div>
</section>
<section id="top-5-especies-barras-apiladas" class="level3">
<h3 class="anchored" data-anchor-id="top-5-especies-barras-apiladas">Top 5 Especies, barras apiladas</h3>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Top%205%20Especies,%20barras%20apiladas-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="mock" class="level3">
<h3 class="anchored" data-anchor-id="mock">Mock</h3>
<p>La muestra MOCK es un control positivo que permite validar la eficiencia y precisión del pipeline de metabarcoding. Analizar MOCK ayuda a detectar posibles errores, contaminaciones o sesgos en la identificación de especies, asegurando la confiabilidad de los resultados.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Mock-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="diversidad-alfa-especies-observadas-por-muestra" class="level3">
<h3 class="anchored" data-anchor-id="diversidad-alfa-especies-observadas-por-muestra">Diversidad alfa: especies observadas por muestra</h3>
<p>La diversidad alfa nos da una medida de la riqueza de especies en cada muestra, permitiendo comparar sitios y detectar patrones de biodiversidad local. Es fundamental para entender la estructura interna de las comunidades y para identificar sitios de alta o baja diversidad.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Diversidad%20alfa-%20especies%20observadas%20por%20muestra-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="análisis-de-similitud-entre-sitios-anosim" class="level3">
<h3 class="anchored" data-anchor-id="análisis-de-similitud-entre-sitios-anosim">Análisis de similitud entre sitios (ANOSIM)</h3>
<p>ANOSIM permite evaluar estadísticamente si las diferencias entre grupos de muestras (por ejemplo, regiones) son significativas. Esto ayuda a validar si los patrones observados en los gráficos reflejan diferencias reales en la composición de las comunidades.</p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
vegan::anosim(x = dist_jac, grouping = meta_filt$region_gc) 
Dissimilarity: jaccard 

ANOSIM statistic R: -0.006727 
      Significance: 0.491 

Permutation: free
Number of permutations: 999

Upper quantiles of permutations (null model):
  90%   95% 97.5%   99% 
0.130 0.178 0.227 0.273 

Dissimilarity ranks between and within classes:
        0%  25% 50%   75%  100%   N
Between  3 59.5 116 180.0 224.5 105
Centro   1 59.5 122 174.5 231.0 105
Norte    2 46.5 116 163.0 228.0  21</code></pre>
</div>
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Análisis%20de%20similitud%20entre%20sitios%20(ANOSIM)-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="análisis-de-varianza-permutacional-permanova" class="level3">
<h3 class="anchored" data-anchor-id="análisis-de-varianza-permutacional-permanova">Análisis de varianza permutacional (PERMANOVA)</h3>
<p>PERMANOVA permite probar hipótesis sobre el efecto de variables ambientales o categóricas en la composición de las comunidades. Es una herramienta robusta para analizar datos ecológicos complejos y para identificar factores que estructuran la biodiversidad.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>permanova_res <span class="ot">&lt;-</span> vegan<span class="sc">::</span><span class="fu">adonis2</span>(comm_filt <span class="sc">~</span> region_gc, <span class="at">data =</span> meta_filt, <span class="at">permutations =</span> <span class="dv">999</span>, <span class="at">method =</span> <span class="st">"bray"</span>)</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(permanova_res)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Permutation test for adonis under reduced model
Permutation: free
Number of permutations: 999

vegan::adonis2(formula = comm_filt ~ region_gc, data = meta_filt, permutations = 999, method = "bray")
         Df SumOfSqs      R2      F Pr(&gt;F)
Model     1  0.03547 0.05623 1.1915  0.335
Residual 20  0.59540 0.94377              
Total    21  0.63087 1.00000              </code></pre>
</div>
</div>
</section>
<section id="non-metric-multidimensional-scaling-nmds" class="level3">
<h3 class="anchored" data-anchor-id="non-metric-multidimensional-scaling-nmds">Non-metric Multidimensional Scaling (NMDS)</h3>
<p>La diversidad beta analiza las diferencias entre comunidades, ayudando a identificar agrupamientos, gradientes ecológicos o efectos de factores ambientales. El NMDS es una herramienta visual poderosa para explorar la estructura de las comunidades y detectar patrones de similitud o diferenciación entre sitios.</p>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Run 0 stress 0.1370224 
Run 1 stress 0.1370225 
... Procrustes: rmse 0.0002112416  max resid 0.0005711191 
... Similar to previous best
Run 2 stress 0.1370223 
... New best solution
... Procrustes: rmse 0.0001128872  max resid 0.00021009 
... Similar to previous best
Run 3 stress 0.1370223 
... Procrustes: rmse 7.125925e-05  max resid 0.0001617045 
... Similar to previous best
Run 4 stress 0.1370224 
... Procrustes: rmse 0.0001753017  max resid 0.0004911779 
... Similar to previous best
Run 5 stress 0.1594096 
Run 6 stress 0.1370225 
... Procrustes: rmse 0.0002382554  max resid 0.0006731628 
... Similar to previous best
Run 7 stress 0.1370224 
... Procrustes: rmse 0.0001898288  max resid 0.0005352449 
... Similar to previous best
Run 8 stress 0.1370223 
... New best solution
... Procrustes: rmse 1.748514e-05  max resid 4.014091e-05 
... Similar to previous best
Run 9 stress 0.1370223 
... Procrustes: rmse 2.040704e-05  max resid 4.589598e-05 
... Similar to previous best
Run 10 stress 0.1594074 
Run 11 stress 0.1370223 
... Procrustes: rmse 1.330126e-05  max resid 3.13536e-05 
... Similar to previous best
Run 12 stress 0.1370223 
... Procrustes: rmse 4.364973e-05  max resid 7.371235e-05 
... Similar to previous best
Run 13 stress 0.1370223 
... Procrustes: rmse 2.013902e-05  max resid 3.640198e-05 
... Similar to previous best
Run 14 stress 0.1370224 
... Procrustes: rmse 0.0001365006  max resid 0.0003798875 
... Similar to previous best
Run 15 stress 0.1370224 
... Procrustes: rmse 0.0001047544  max resid 0.0002491128 
... Similar to previous best
Run 16 stress 0.1544301 
Run 17 stress 0.1370223 
... Procrustes: rmse 1.581079e-05  max resid 3.413792e-05 
... Similar to previous best
Run 18 stress 0.1370225 
... Procrustes: rmse 0.0002410391  max resid 0.0006522875 
... Similar to previous best
Run 19 stress 0.1370224 
... Procrustes: rmse 0.0001795064  max resid 0.0005080207 
... Similar to previous best
Run 20 stress 0.1370223 
... Procrustes: rmse 8.749444e-05  max resid 0.0002478001 
... Similar to previous best
*** Best solution repeated 11 times</code></pre>
</div>
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Diversidad%20beta-%20matriz%20de%20distancias%20y%20NMDS%20-1.png" class="img-fluid" width="768"></p>
</div>
</div>
</section>
<section id="enfoque-libre-de-taxonomia" class="level3">
<h3 class="anchored" data-anchor-id="enfoque-libre-de-taxonomia">Enfoque libre de taxonomia</h3>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Run 0 stress 0.03347727 
Run 1 stress 0.04688075 
Run 2 stress 0.03347751 
... Procrustes: rmse 0.0006743258  max resid 0.001767105 
... Similar to previous best
Run 3 stress 0.03347719 
... New best solution
... Procrustes: rmse 0.0005258878  max resid 0.001380117 
... Similar to previous best
Run 4 stress 0.04689176 
Run 5 stress 0.0334773 
... Procrustes: rmse 6.29983e-05  max resid 0.0001651465 
... Similar to previous best
Run 6 stress 0.03347702 
... New best solution
... Procrustes: rmse 0.0001927886  max resid 0.0005921515 
... Similar to previous best
Run 7 stress 0.05602194 
Run 8 stress 0.03347722 
... Procrustes: rmse 0.0002106482  max resid 0.000652404 
... Similar to previous best
Run 9 stress 0.03348639 
... Procrustes: rmse 0.001728899  max resid 0.005843138 
... Similar to previous best
Run 10 stress 0.03720742 
Run 11 stress 0.04755698 
Run 12 stress 0.03720781 
Run 13 stress 0.03777817 
Run 14 stress 0.03347739 
... Procrustes: rmse 0.0004068159  max resid 0.001139606 
... Similar to previous best
Run 15 stress 0.03778058 
Run 16 stress 0.0334771 
... Procrustes: rmse 0.000223167  max resid 0.0006002065 
... Similar to previous best
Run 17 stress 0.03347721 
... Procrustes: rmse 0.0002066242  max resid 0.0006345077 
... Similar to previous best
Run 18 stress 0.03347718 
... Procrustes: rmse 0.0002103211  max resid 0.0005831739 
... Similar to previous best
Run 19 stress 0.03348894 
... Procrustes: rmse 0.00201916  max resid 0.006365908 
... Similar to previous best
Run 20 stress 0.03347718 
... Procrustes: rmse 0.0001629838  max resid 0.000484351 
... Similar to previous best
*** Best solution repeated 9 times</code></pre>
</div>
<div class="cell-output-display">
<p><img src="Report_files/figure-html/Enfoque%20libre%20de%20taxonomia-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="anosim" class="level3">
<h3 class="anchored" data-anchor-id="anosim">ANOSIM</h3>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
vegan::anosim(x = dist_asv, grouping = nmds_asv_df$region_gc) 
Dissimilarity: jaccard 

ANOSIM statistic R: 0.2231 
      Significance: 0.037 

Permutation: free
Number of permutations: 999

Upper quantiles of permutations (null model):
  90%   95% 97.5%   99% 
0.148 0.194 0.249 0.349 

Dissimilarity ranks between and within classes:
        0% 25% 50% 75% 100%   N
Between  1  67 132 189  231 105
Centro   2  41  96 145  222 105
Norte   36  88 152 194  217  21</code></pre>
</div>
<div class="cell-output-display">
<p><img src="Report_files/figure-html/ANOSIM%20-%20ASV%20-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>
<section id="análisis-de-varianza-permutacional-permanova---asv" class="level3">
<h3 class="anchored" data-anchor-id="análisis-de-varianza-permutacional-permanova---asv">Análisis de varianza permutacional (PERMANOVA) - ASV</h3>
<div class="cell">
<div class="cell-output cell-output-stdout">
<pre><code>Permutation test for adonis under reduced model
Permutation: free
Number of permutations: 999

vegan::adonis2(formula = comm_filt ~ region_gc, data = nmds_asv_df, permutations = 999, method = "jaccard")
         Df SumOfSqs      R2      F Pr(&gt;F)
Model     1  0.08989 0.05633 1.1939  0.299
Residual 20  1.50573 0.94367              
Total    21  1.59562 1.00000              </code></pre>
</div>
</div>
</section>
<section id="gráfico-nmds-final---asv" class="level3">
<h3 class="anchored" data-anchor-id="gráfico-nmds-final---asv">Gráfico NMDS final - ASV</h3>
<p>Este gráfico muestra la estructura de las comunidades de peces en el Golfo de California, comparando dos regiones (“Centro” y “Norte”) mediante un análisis NMDS basado en la distancia Jaccard.</p>
<ul>
<li><p>Cada punto representa una muestra (sitio) y su posición refleja la composición de especies: sitios más cercanos tienen comunidades más similares.</p></li>
<li><p>Los colores y elipses agrupan las muestras por región (“Centro” en verde y “Norte” en morado), mostrando la tendencia de agrupamiento regional.</p></li>
<li><p>Las etiquetas identifican cada sitio de muestreo.</p></li>
<li><p>Valores estadísticos en la esquina inferior izquierda:</p>
<ul>
<li><p>PERMANOVA: F = 1.19, R² = 0.06, p = 0.27. Indica que no hay diferencias significativas en la composición de especies entre regiones (p &gt; 0.05).</p></li>
<li><p>ANOSIM: R = 0.22, p = 0.043. Indica una diferencia significativa pero pequeña entre regiones (p &lt; 0.05).</p></li>
<li><p>Stress: 0.033. Un valor bajo, lo que significa que la representación gráfica es fiel a las distancias reales entre comunidades.</p></li>
</ul></li>
</ul>
<p>El gráfico sugiere que hay una ligera diferencia en la composición de especies entre las regiones “Centro” y “Norte”, respaldada por ANOSIM pero no por PERMANOVA. La agrupación por región es visible, pero las comunidades no están completamente separadas.</p>
<div class="cell">
<div class="cell-output-display">
<p><img src="Report_files/figure-html/plot%20final%20NMDS%20-%20ASV-1.png" class="img-fluid" width="672"></p>
</div>
</div>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>